src/main/java/org/broadinstitute/dsm/export/ExportToES.java

```diff
@@ -41,7 +40,21 @@
     public static final String SELF = "SELF";
     public static final String FAMILY_ID = "FAMILY_ID";
 
-    public static void exportObjectsToES(ExportPayload payload) {
+    public static class ExportPayload {
+        private String index;
+        private String study;
+
+        public String getIndex() {
+            return index;
+        }
+
+        public String getStudy() {
+            return study;
+        }
+    }
+
+    public static void exportObjectsToES(String data) {
+        ExportPayload payload = gson.fromJson(data, ExportPayload.class);
         int instanceId = ddpInstanceDao.getDDPInstanceIdByGuid(payload.getStudy());
 
         logger.info("Started exporting workflows for instance with id " + instanceId);
```

Comment:
Putting logger messages into related method should be more useful than wrapping method in loggers.